cogs/battle.py

```diff
@@ -0,0 +1,111 @@
+import discord
+from discord.ext import commands
+from discord.ui import Select, View
+import random
+
+
+class battleSystem(commands.Cog):
```

Comment:
```suggestion
class BattleSystem(commands.Cog):
```
use TitleCase convention for class names